src/app/shared/styles/my-entry-sidebar.scss

```diff
@@ -0,0 +1,102 @@
+@use '@angular/material' as mat;
+
+@import 'materialColorScheme.scss';
+
+// Grouping titles by registery/source control
+div.registryTitle,
+div.sourceTitle {
+  font-size: 14px;
+  font-weight: 500;
+  text-align: center;
+  margin-top: 2.8rem;
+}
+
+// Header style
+mat-expansion-panel-header,
+mat-expansion-panel-header[aria-expanded='true'] {
+  height: 3.4rem;
+  padding-left: 1rem;
+  padding-right: 1rem;
+}
+
+// Header style - Expanded
+mat-expansion-panel-header,
+mat-expansion-panel-header[aria-expanded='true']:hover,
+mat-expansion-panel-header[aria-expanded='true']:focus {
+  font-size: 14px;
+  font-weight: 500;
+}
+
+// Header style - Collapsed
+mat-expansion-panel-header[aria-expanded='false'] {
+  color: #487980;
+  text-decoration: underline;
+  background-color: white;
+
+  :hover {
+    background-color: mat.get-color-from-palette($dockstore-app-gray, 5);
+  }
+}
+
+// Expansion body expanded
+:host ::ng-deep mat-expansion-panel .mat-expansion-panel-content .mat-expansion-panel-body {
+  padding: 0rem 0.8rem 1.2rem;
+}
+
+// Expansion panel outlines
+mat-expansion-panel {
+  margin: 0px 2px;
+  border-bottom: 0.1rem solid mat.get-color-from-palette($dockstore-app-gray, 3);
+}
+
+//Avoid overlap with borders (only first panel gets top, all others get bottom)
+.mat-accordion .mat-expansion-panel:first-of-type {
+  border-top: 0.1rem solid mat.get-color-from-palette($dockstore-app-gray, 3);
+}
+
+// Expansion panel corners
+.mat-accordion .mat-expansion-panel:last-of-type,
+.mat-accordion .mat-expansion-panel:first-of-type {
+  border-radius: 0rem;
+}
+
+// Refresh button div
+.mat-action-row {
+  border-color: mat.get-color-from-palette($dockstore-app-gray, 3);
+  padding: 0rem 0rem 0.3rem 0rem;
+}
+
+////////// MAT TABS ///////////
+:host ::ng-deep .mat-tab-label-content {
+  font-size: 12px;
+}
+
+// Selected tab style
+:host ::ng-deep .mat-tab-label-active {
+  opacity: 1;
+
+  .mat-tab-label-content {
+    font-weight: bold;
+    color: #181f40;
+  }
+}
+
+// Selected entry style
+.selected,
```

Comment:
What is `selected` used for? Does it have the same styles as `selectedGitHubAppTool`? If so, could you collapse it into one class, like `selectedTool`?